components/Workshop/WorkshopListingBodyLocation.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Flex, Text } from "@chakra-ui/react";
+import React from "react";
+import { BsCameraVideo, BsPinMap } from "react-icons/bs";
+import { Workshop } from "../../shared/schemas";
+
+interface IProps {
+  workshop: Workshop;
+}
+
+export default function WorkshopListingBody({ workshop }: IProps) {
```

Comment:
I think this probably needs renaming to WorkshopListingBodyLocation?

Reply:
Well spotted, fixed.